psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -120,8 +120,12 @@ def initialize_jk(self, memory, jk=None):
     jk.set_memory(memory)
     jk.set_do_K(functional.is_x_hybrid())
     jk.set_do_wK(functional.is_x_lrc())
+    jk.set_wcombine(core.get_option("SCF","WCOMBINE"))
```

Comment:
can this be set upon initializing the class instance?

Reply:
Pytests pass locally with that line removed, but I saw line 122 (`jk.set_do_wK(functional.is_x_lrc())`) in the code before starting this project, so I thought it would make sense to leave the option here out of an abundance of caution and to match the pre existing protocol.

Reply:
it's good to keep all the options handling in one spot (back c-side in this case). `jk`, `functional`, `memory` are all local to the function or local to the class; whereas, the `get_options` call to to globals.

Reply:
Latest commit makes the suggested change